opm/parser/eclipse/EclipseState/Schedule/MSW/SegmentSet.cpp

```diff
@@ -21,6 +21,11 @@
 #include <cmath>
 #include <map>
 
+#ifdef _WIN32
+#define _USE_MATH_DEFINES 
+#include <math.h>
```

Comment:
It's a Windows thing. It uses that to enable/disable parts of the `math.h` file. Regardless, there are better ways of solving this.